FTB/Signatures/CrashInfo.py

```diff
@@ -285,7 +286,7 @@ def fromRawCrashData(stdout, stderr, configuration, auxCrashData=None, cacheObje
             if ubsanString in line and re.match(ubsanRegex, line) is not None:
                 result = UBSanCrashInfo(stdout, stderr, configuration, auxCrashData)
                 break
-            elif asanString in line:
+            elif asanString in line or ubsanString2 in line:
                 result = ASanCrashInfo(stdout, stderr, configuration, auxCrashData)
```

Comment:
I don't like that we're handling ubsan crashes with the asan class when we have an ubsan crash. I realize the format is identical. I wonder if it's time for `xSanCrashInfo`

Reply:
OK I opened issue #615 to keep track of it.